resources/views/messages/error.blade.php

```diff
@@ -0,0 +1,11 @@
+
+@if ($errors->any())
+        <div class="alert alert-danger">
+            <ul>
+                @foreach ($errors->all() as $error)
+                <li>{{ $error }}</li>
+                @endforeach
+            </ul>
+        </div>
+        <br /> 
+@endif
```

Comment:
```suggestion
@if ($errors->any())
    <div class="alert alert-danger">
        <ul>
            @foreach ($errors->all() as $error)
            <li>{{ $error }}</li>
            @endforeach
        </ul>
    </div>
    <br /> 
@endif
```
tab nhiều quá rồi a :v indent 4 spaces.

Reply:
resolved